students/ryba_paula/lesson_02_flow_control/lost_card.py

```diff
@@ -0,0 +1,12 @@
+cards = int(input())
+
+sum_of_cards = 0
+remaining_cards = 0
+
+for i in range(1, cards + 1):
+    sum_of_cards += i
+
+for i in range(1, cards):
+    remaining_cards += int(input())
```

Comment:
można to z optymalizować i zamiast liczyć sumę wartości wprowadzanych kart, można je odejmować od oczekiwanej sumy i na końcu zostanie brakująca karta